api/db/db_models.py

```
@@ -60,18 +69,96 @@
+        return super().db_value(value)
+
+    def python_value(self, value):
+        if is_gaussdb_compatible_database() and value is None:
+            # GaussDB 适配点：调用方仍按历史约定看到 ""，不需要在业务代码里到处
+            # 增加 `is None` 分支；隔离逻辑收敛在字段类型本身。
+            return ""
+        return super().python_value(value)
+
+    def __eq__(self, rhs):
+        if is_gaussdb_compatible_database() and rhs == "":
+            # GaussDB A-compatible 下 `field = ''` 实际等价于 `field = NULL`，
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)